service/routes/docs.js

```diff
@@ -297,12 +299,10 @@ function route(req, res, next) {
 		.then(locals => {
 			template(locals.pageName)
 				.then(templFn => res.send(templFn(locals)))
-				.catch(function() { next(); })
+				.catch(() => next())
```

Comment:
I think we can pass `next` in to catch directly rather than wrapping in another function.

`.catch(next)`